libraries/AP_Mount/STorM32_lib.cpp

```diff
@@ -0,0 +1,404 @@
+#include <GCS_MAVLink/include/mavlink/v2.0/checksum.h>
+#include <AP_Notify/AP_Notify.h>
+#include <AP_Mount/STorM32_lib.h>
+
+//******************************************************
+// STorM32_lib class functions
+//******************************************************
+
+/// Constructor
+STorM32_lib::STorM32_lib() :
+    _serial_is_initialised(false),
+    _storm32link_seq(0)
+{
+    _serial_in.state = SERIALSTATE_IDLE;
```

Comment:
Same comments: zero initialization isn't needed and this one can be put in class declaration.